extensions/micrometer/runtime/src/main/java/io/quarkus/micrometer/runtime/export/exemplars/OpenTelemetryExemplarContextUnwrapper.java

```diff
@@ -0,0 +1,35 @@
+package io.quarkus.micrometer.runtime.export.exemplars;
+
+import java.util.function.Function;
+
+import jakarta.enterprise.context.Dependent;
+
+import io.opentelemetry.context.Context;
+import io.opentelemetry.context.Scope;
+import io.quarkus.opentelemetry.runtime.QuarkusContextStorage;
+
+@Dependent
+public class OpenTelemetryExemplarContextUnwrapper implements OpenTelemetryContextUnwrapper {
+
+    @Override
+    public <P, R> R executeInContext(Function<P, R> methodReference, P parameter, io.vertx.core.Context requestContext) {
```

Comment:
is requestContext a duplicated context? If it's a root context that can be problematic.

Reply:
It should be, it's the context bound to a particular HTTP Request